### PR TITLE
Created the SOFT3 Wiki page

### PR DESCRIPTION
**Pull request by WalterToro**
### Description

Created wiki page for SOFT3 community tournament, tournament is ongoing so info will be updated every weekend.

### Status

Pending Review

### Additional Notes

@juankristal should review this before merging.

### CODE REVIEW

**Comment by Hydria** [comment]
3rd times the charm! > third time's the charm
You can enter from any country, with any rank or performance points. (sounds weird in phrasing, would probably reword it to) You can enter from any country, **at** any rank **with any amount of** performance points.
To be determinated > To be determined
Discussion Thread (capitalise the T)
Note: This list is ordered by **his** overall seed number (remove his)
The results of the qualifiers can be found on this link. > The results of the qualifiers can be found **via** this link.
however, participants cut down to a power of 2. > however, participants **are** cut down to a power of 2.
If 33 players enter, the lowest individual gets eliminated and the top 32 **go on**. > move on (better choice of words, same for all similar wording)
All players will have two chances at each qualifier, > All players will have two chances at each qualifier **map**,
where the highest of the two runs on each qualifier > where the highest of the two runs on each qualifier **map**
Players will be organized in groups based on their availability. > Players will be organized **into** groups based on their availability.
If this is a confusing process, do not worry **if you do not understand** (remove bold, superfluous text)
The purpose of putting you **guys** in groups of 8 > The purpose of putting you in groups of 8 (gender pronouns offend people)
The score system we will be using > The **scoring** system we will be using
The total of the four songs played will accumulate to your final qualification score. > The total of the four songs played will **cumulate** to your final qualification score. (changed to cumulate for consistency throughout)
will be placed with 3 different players > will be placed with 3 **additional** players
After the first qualification round, the remaining 128 (missed comma)
and they won't play  (missed apostrophe)
the second round of qualification or post-qualification stage**s** (pluralize)
This teams will be the top seeded teams of the group in all the cases. (???????)
Those who get this seed**s** (remove pluralization)
will get guaranteed High-Seed in the group stage > will **be** guaranteed **the** High-Seed **spot** in the group stage
This players will > **These** players will
depending on who ends up qualified, their seed of the qualification stage (added comma)
and **it** the sorting (remove it)
The groups will be made of 4 players each and they will be a number of 16 groups > The groups will be made of 4 players each and **there** will be 16 groups (removing more superfluous text)
and because of that, streaming every single map (added comma)
every single map is just **literally** not doable. (we're professional, we don't use "literally")
Even still > Even then (sounds better)
If it is required the match schedules will consider running them for more time (this makes no sense)
I (juankristal) will have sole (bit late to be introducing yourself, should probably be done at) otherwise noted by myself.
and **just** be respectful. (professionalism, no just)
Warm-up songs will be on free mode > Free Mod (actual name of the setting/consistency)
In early stages your Warm-up cant be longer than 4 minutes. > In **the** early stages, your Warm-up cant be longer than 4 minutes. (added comma)
the other player will pick songs > the other player will pick **a song**

**Comment by WalterToro** [comment]
Changes applied.
